.github/CODEOWNERS

```
@@ -1,3 +1 @@
-*.md @jessicaschilling
-/add-on/_locales/en/messages.json @jessicaschilling
-* @lidel
+* @lidel @meandavejustice @ipfs/gui-dev
```

@meandavejustice do you want to get pinged for reviews on everything, or only about V3 version  from https://github.com/ipfs/ipfs-companion/pull/1078?

```suggestion
* @lidel @ipfs/gui-dev
/mv3 @meandavejustice
```

---

leaving as is after @meandavejustice 's response